### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by icristescu**

### CODE REVIEW

**Comment by icristescu** [comment]
The ci fails with 
```
-  ERROR: (Invalid_argument "Irmin.head: no head")
+  ERROR: Invalid_argument("Irmin.head: no head")
```
the culprit seems to be `ocamlformat`? I can reproduce this error only if I uninstall `ocamlformat` from my switch.

**Review by maiste — APPROVED** [review]

**Comment by zshipko** [comment]
I ended up doing this: https://github.com/mirage/irmin/pull/1750/commits/73392419e52c7bcb8c09f0946df632380f43d456 - since the output is different on my machine compared to CI

File: test/irmin-unix/test_command_line.t [hunk 1]
```diff
@@ -86,7 +86,7 @@ Check mismatched hash function
   $ irmin set --root ./test-hash -s irf -h sha1 abc 123
```
Comment (tomjridge) [inline test/irmin-unix/test_command_line.t:86]: For the above line, perhaps change to: "Check mismatched hash function. NOTE the format of the error message changes when ocamlformat is installed in the switch; the following assumes ocamlformat IS installed."

**Comment by tomjridge** [comment]
I guess @zshipko's suggested change is probably the best we can hope for if the formatting of error messages can change. Depending on the exact way an exception is represented as a string is going to be very fragile anyway.

**Comment by icristescu** [comment]
> I ended up doing this: [7339241](https://github.com/mirage/irmin/commit/73392419e52c7bcb8c09f0946df632380f43d456) - since the output is different on my machine compared to CI

we can just remove the test then, no?

**Comment by tomjridge** [comment]
Don't we still want the test, and to check that it is indeed an errror? @zshipko 's patch still executes the test and checks for an error, just not the exact error string that results.

**Comment by icristescu** [comment]
thanks, I added the patch and I'll merge this once the CI is green

**Comment by tomjridge** [comment]
Yeah, looks good.

**Pull request merged**